tests/fw_cli/test_commands.py

```diff
@@ -0,0 +1,160 @@
+from fw_pipeline_io.fw_cli.commands import FWCLI
+from unittest.mock import MagicMock, patch
+import unittest
+
+
+class TestFWCLI(unittest.TestCase):
```

Comment:
Most of these tests are self-explanatory from the function name, but it would be nice to have a brief, one-sentence doc string stating what each test is checking for.

Reply:
added docstrings